### DISCUSSION TITLE
New release ?

### DISCUSSION THREAD

**Discussion by gwbres** [nyx-space/hifitime#275] [Q&A] [answered]
Hello, would you care in releasing the latest improvements ? so I can update my tools

**Comment by ChristopherRabotin** [comment]
The release process just started. I'd forgotten to create it last night, sorry.

**Reply by ChristopherRabotin** [reply, marked as answer]
It's live!

Rust: https://crates.io/crates/hifitime
Python: https://pypi.org/project/hifitime/

**Reply by gwbres** [reply]
no hurries don't worry ; )
I will try to update my tools tonight, I will also keep you updated if i see any other thing
